test/compilable/imports/imp16085b.d

```diff
@@ -0,0 +1,17 @@
+import imp16085 : S;
+
```

Comment:
Maybe add a test for alias lambda (e.g. `alias templateLiteralAndTemplate = x => 1;`).